commands/version.go

```diff
@@ -8,12 +8,11 @@ import (
 	"runtime"
 
 	"github.com/morikuni/aec"
+	"github.com/openfaas/faas-cli/version"
 	"github.com/spf13/cobra"
 )
 
 // GitCommit injected at build-time
```

Comment:
This should have been removed

Reply:
why would it be removed? Without it the version command cannot use the new package.

Reply:
I think I understand this today lol. I've removed the comment from the merge in-case it's just GitHub displaying poorly. The import has to be there (for some reason GitHub highlights that), the comment is gone.